### PR TITLE
update libs

### PR DESCRIPTION
**Pull request by tbo47**
small update

#### **Verify that...**

- [x] `npm test` passes
- [x] `npm run api` passes
- [x] `npm run lint` passes
- [x] `npm run build` still works

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,32 +1,32 @@
 {
```
Comment (Zemledelec) [inline package-lock.json:1]: Are we sure that we need lock file in repo?

FYI @pvlt

**Review by Zemledelec — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (pavletto) [reply package-lock.json:1]: Yep, we need it. lockfile ensures that we all use the same dependencies and optimize dependencies installation in ci. [package-lock.json](https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json)

**Review by pavletto — COMMENTED** [review]
@tbo47, please, add `.vscode` dir to `.gitignore` files

**Comment by tbo47** [comment]
> 

done!

**Review by pavletto — APPROVED** [review]

**Pull request merged**